bin/compile

```diff
@@ -35,6 +37,10 @@ fi
 if [ -f $BUILD_DIR/.datadog/dogstatsd ]; then
   chmod +x $BUILD_DIR/.datadog/dogstatsd
 fi
+
+chmod +x $BUILD_DIR/.datadog/scripts/update_agent_config.sh
+chmod +x $BUILD_DIR/.datadog/scripts/create_logs_config.py
+chmod +x $BUILD_DIR/.datadog/scripts/get_tags.py
```

Comment:
We don't need these lines, right? 
```suggestion
```

Reply:
Yes, good catch! I forgot to remove these lines